src/Raven.Studio/typescript/models/wizard/domainInfo.ts

```diff
@@ -57,22 +57,31 @@ class domainInfo {
                                                     });
                              };
         
+        const rg1 = /^[a-zA-Z0-9-]+$/;
+        const rg2 = /^(?!-)(?!.*-$)/;
```

Comment:
can we use startsWith endsWith instead of regexp ? ;-)

Reply:
done :)